tools/change-registry.sh

```diff
@@ -87,6 +88,11 @@ function parse_args() {
   REGISTRY_SCHEME="$(sed "s/:\/\///" <<< "$REGISTRY_SCHEME")"
   DECKHOUSE_TAG="$(kubectl -n d8-system get deploy deckhouse -o json | jq '.spec.template.spec.containers[] | select(.name=="deckhouse") | .image | split(":")[-1]' -r)"
 
+  tmp_external_modules_tag="$(kubectl -n d8-system get deploy deckhouse -o json | jq '.spec.template.spec.initContainers[] | select(.name=="init-external-modules") | .image | split(":")[-1]' -r 2>/dev/null)"
+  if [[ "$tmp_external_modules_tag" != "" ]]; then
+    DECKHOUSE_EXTERNAL_MODULES_IMAGE_TAG="${tmp_external_modules_tag}"
```

Comment:
i think it will be DECKHOUSE_INIT_EXTERNAL_MODULES_CONTAINER

Reply:
Maybe it would be better if we make it like the deckhouse image var name on line 89? something like `DECKHOUSE_INIT_EXTERNAL_MODULES_CONTAINER _TAG`?